_docs/reference/labels.md

```diff
@@ -44,10 +44,20 @@ To add overcommit labels to a pool:
 ```bash
 storageos pool update --label-add storageos.com/overcommit=20 default
 ```
-This would result in being able to create volumes whose total capacity exceeds
-the capacity of the underlying node by 20%. It is not possible to create a 
-volume whose capacity exceeds the capacity of underlying nodes as StorageOS
-volumes must always be able to fit on an underlying node. 
+This would result in the total pool volume being increased by 20% and the
+maximum possible volume size being increased by 20%. If you have 120GB pool
```

Comment:
maximum possible volume size being --> maximum possible volume size **per node** being

Reply:
I believe I've resolved this.